src-doc/contenuti/use-cases/uc21.tex

```diff
@@ -0,0 +1,11 @@
+\section{UC21 - Impostazione area in upper\_state}\label{uc:21}
+\paragraph{Intenzione in contesto} L'attore primario vuole impostare un'area di illuminazione in modalità \textit{upper\_state}
+\paragraph{Attore primario} L'attore primario è il sensore di presenza
+\paragraph{Precondizioni} L'attore primario è autenticato ed autorizzato dal sistema.
+\paragraph{Post-condizioni} L'area di illuminazione selezionata è impostata in modalità \textit{upper\_state}
+\paragraph{Scenario principale}
+\begin{enumerate}
+    \item L'attore primario comunica al sistema a quale area vorrebbe impostare l'\textit{upper\_state};
+    \item l'attore primario imposta l'area in \textit{upper\_state};
+    \item il sistema comunica all'attore primario la conferma dell'impostazione in upper state dell'area di gestione.
```

Comment:
Suggerimento:
```suggestion
    \item il sistema comunica all'attore primario la conferma dell'impostazione in \textit{upper\_state} dell'area di gestione.
```

Reply:
Qui avevo messo in italic perchè è uno stato che ha "un nome". Cosa consigli di fare per documentare poi lo stato?

Reply:
ah ok, ora ho riletto con calma, avevo letto lo avessi tolto non so perché